conans/cli/api/subapi/profiles.py

```diff
@@ -26,6 +26,16 @@ def get_default_build(self):
         loader = ProfileLoader(self._cache)
         return loader.get_default_build()
 
+    def get_profiles(self, args):
```

Comment:
This doens't seem very good. This is very cli-args specific, having something in the API layer to parse CLI  args is not a good pattern.

It is totally fine to have reusable pieces in the command layer, that parse args, or add common args. But all the functionality mapping from cli args to domain variables should belong to the command layer, and the API should receive regular variables.